lesson08/script.js

```diff
@@ -0,0 +1,44 @@
+function createLoading(){
+    const loadingImg = document.createElement('img');
+    loadingImg.src = "loading-circle.gif";
+    loadingImg.alt = "ローディング画像";
+    document.getElementById('js-loading').appendChild(loadingImg);
```

Comment:
Question.
Why did you stop defining variables? If you have any specific reason, I'd like to hear it.

Reply:
Because document.getElementById ('js-loading') is not used repeatedly in this function.
I referred to comments by Sae-san and Mr.Moriken about Ayaka-san's commit.
[Link](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/8#discussion_r776686256)